malfeasance/handler.go

```diff
@@ -391,6 +391,10 @@ func validateInvalidPostIndex(
 	proof *wire.InvalidPostIndexProof,
 ) (types.NodeID, error) {
 	atx := &proof.Atx
+	if err := checkIdentityExists(db, atx.SmesherID); err != nil {
```

Comment:
The naming of this function is a little bit confusing given the current DB structure.
We have `identities` table which is actually used to store the malfeasance proofs (maybe a bit of misnomer...); and here at the first glance it might appear that we're checking for a row in that table while actually we're about to create one.
Maybe we should rename this function to `HaveATXs` or something else?

Reply:
renamed to `hasPublishedAtxs`